src/skill.cpp

```diff
@@ -221,6 +228,11 @@ void SkillLevel::readBook(int minimumGain, int maximumGain, int maximumLevel)
     practice();
 }
 
+bool SkillLevel::can_train() const
```

Comment:
Looks like this should be a static function, or maybe even in the `Skill` class. Or do you plan to extend it using non-static members of  `SkillLevel`?